libnd4j/include/ops/declarable/helpers/cuda/image_resize.cu

```diff
@@ -1115,8 +1115,9 @@ namespace helpers {
         auto err = cudaMalloc(&pSt, sizeof(ImageResizerState));
         err = cudaMemcpyAsync(pSt, &st, sizeof(ImageResizerState), cudaMemcpyHostToDevice, *stream);
         ScaleCache<T>* cachePool;
-        err = cudaMalloc(&cachePool, sizeof(ScaleCache<T>) * st.batchSize * st.outWidth * st.outHeight);
-        resizeAreaKernel<T><<<128, 2, 2048, *stream>>>(pSt, cache, scale, inputPtr, input->specialShapeInfo(), outputPtr,
+        auto cachePoolSize = sizeof(ScaleCache<T>) * st.batchSize * st.outWidth * st.outHeight;
+        err = cudaMalloc(&cachePool, cachePoolSize);
```

Comment:
Okay. You've got error code. Great. But where's validation of `err`? What if allocation failed?